templates/Web/Pages/Vue.Grid/src/components/GridComponent.vue

```diff
@@ -1,9 +1,9 @@
 ﻿<template>
-  <div class="col-md-4 col-sm-12 p-5">
+  <b-col sm="12" md="4" class="p-5">
     <img src="@/assets/GreyBox.svg" alt="Default Grey Box" class="mb-3" />
     <h3>{{header}}</h3>
     <p>{{description}}</p>
-  </div>
+  </b-col>
```

Comment:
Just out of curiosity, what does `b-col` signify here?

Reply:
Here I am wrapping a grid component (individual grid on the grid page) into `<b-col>`. It is Bootstrap-Vue style tag for grid layout. Here is a screenshot of the example provided by Bootstrap-Vue Docs.
<img width="795" alt="Screen Shot 2019-08-07 at 8 11 36 PM" src="https://user-images.githubusercontent.com/12874317/62672474-b4e56c00-b94f-11e9-83b9-379c1dbc753f.png">